2.Nomic_nELISA_Analysis/1.umap/PBMC/scripts/PBMC_umap.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import toml
+import umap
+
+# In[2]:
+
+
+nELISA_plate_430420_PBMC_path = pathlib.Path(
+    "../../Data/clean/Plate2/nELISA_plate_430420_PBMC_cleanup4correlation.csv"
+)
+manual_cluster_1_path = pathlib.Path(
+    "../../Data/clean/Plate2/Manual_Treatment_Clusters_1.csv"
+)
+
+manual_cluster_2_path = pathlib.Path(
+    "../../Data/clean/Plate2/Manual_Treatment_Clusters_2.csv"
+)
+
+treatment_clusters_path = pathlib.Path(
+    "../../../1.Exploratory_Data_Analysis/utils/params.toml"
+)
+
+
+nELISA_plate_430420_PBMC = pd.read_csv(nELISA_plate_430420_PBMC_path)
+manual_clusters_1 = pd.read_csv(manual_cluster_1_path)
+manual_clusters_2 = pd.read_csv(manual_cluster_2_path)
+treatments = toml.load(treatment_clusters_path)["list_of_treatments"]["treatments"]
+
+nELISA_orgingal_plate = nELISA_plate_430420_PBMC.copy()
+
+
+# In[3]:
+
+
+# select data only columns and make floats
+nELISA_data_values = nELISA_orgingal_plate.filter(like="NSU", axis=1)
+nELISA_data_values = nELISA_data_values.astype("float")
```

Comment:
Could these calls be chained?

```suggestion
nELISA_data_values = nELISA_orgingal_plate.filter(like="NSU", axis=1).astype("float")
```